tendrl/node_agent/alerts/alert_socket.py

```diff
@@ -0,0 +1,68 @@
+from _socket import error as _socket_error
+import gevent.event
+import gevent.greenlet
+from gevent.server import StreamServer
+from io import BlockingIOError
+import logging
+from tendrl.common.alert import AlertUtils
+from tendrl.common.config import ConfigNotFound
+from tendrl.common.config import TendrlConfig
+from tendrl.node_agent.alerts.base_alert_handler import AlertHandlerManager
+from tendrl.node_agent.alerts.base_alert_handler import NoHandlerException
+import tendrl.node_agent.manager.utils as utils
+import uuid
+
+config = TendrlConfig()
+LOG = logging.getLogger(__name__)
+RECEIVE_DATA_SIZE = 4096
+
+
+class AlertsManager(gevent.greenlet.Greenlet):
+
+    def read_socket(self, sock, address):
+        try:
+            data = sock.recv(RECEIVE_DATA_SIZE)
+            alert_utils = AlertUtils()
+            alert = alert_utils.validate_alert_json(data)
+            alert['significance'] = 'HIGH'
```

Comment:
Yes this is not configurable.

And as per the plan in https://github.com/Tendrl/specifications/blob/master/specs/pluggable_alert_delivery.adoc --
"Each alert will carry additional attribute that carries its significance which will be added by respective handler. The supported significance levels can be 'HIGH', 'MEDIUM' and 'LOW'.This logic will be hardcoded in the codebase that handles the alerts."